CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - Fixed permissions checks in the UI that prevented users with certain limited permissions from creating report configurations.
 - ROX-9946: Fixed default permissions for the default Vuln Reporter role to exclude the modify permission on notifiers, since it is not needed for report creation.
 - Added AllowPrivilegeEscalation as a new policy criteria.
+- Image signatures are now supported throughout the system, specifically cosign signatures. A new integration type for
```

Comment:
I'm not sure we need to call out new feature in the changelog, they shall be added automatically from jira and by product. As long as there are no changes or deprecations to the API, it's okay to leave out.

Reply:
Ah, that explains why I didn't see any examples 🥲